### DISCUSSION TITLE
Welcome to runpod-worker-comfy 🚀

### DISCUSSION THREAD

**Discussion by TimPietrusky** [runpod-workers/worker-comfyui#10] [Announcements]
Hey everyone👋

We are thrilled to welcome you to our GitHub repository for runpod-worker-comfy. If you're passionate about automating workflows, generating awesome images with stable diffusion, or just love tinkering with Docker and ComfyUI, you're in the right place!

🌟 About runpod-worker-comfy: This project is all about providing a Docker image that integrates seamlessly with RunPod. It helps create a worker environment where ComfyUI can thrive. For those who might be new, ComfyUI is a fantastic tool that leverages stable diffusion techniques to generate images. It's like giving a digital artist a superpowered brush!

🎯 Our Goals: We started this project with a simple aim: to make it easier for anyone to launch and manage image generation workflows. Whether you're a developer, a hobbyist, or somewhere in between, we hope this tool empowers you to create amazing things.

🚀 Getting Started: New here? No worries! You can find detailed instructions on setting everything up in our [README](https://github.com/blib-la/runpod-worker-comfy). It's a step-by-step guide to get you from zero to hero in no time.

🤝 Join the Crew: Your ideas and contributions can help shape the future of runpod-worker-comfy. Found a bug? Have a suggestion? Or maybe you've devised a cool new feature? We'd love to hear about it! Check out our [contribution guidelines](https://github.com/blib-la/runpod-worker-comfy/blob/main/.github/CONTRIBUTING.md) for more details on how to get involved.

📚 Respect and Collaboration: We believe in a community where everyone feels welcome. Please take a moment to review our [Code of Conduct](https://github.com/blib-la/runpod-worker-comfy/blob/main/.github/CODE_OF_CONDUCT.md). Let's keep our discussions friendly, helpful, and constructive.

🔮 What's Next: We're always thinking about what's next for runpod-worker-comfy. And of course, your input on this is invaluable!

❓ Questions or Feedback: Got questions? Need help? Or just want to share your experience using runpod-worker-comfy? Don't hesitate to start a discussion right here. This space is for you to connect, share, and learn from each other.

📩 Stay in Touch: If you need to reach out directly, feel free to send a message. We're here to help!

Once again, welcome to our community. Let's build something amazing together!

Cheers,
Blibla

**Comment by ultimatech-cn** [comment]
I am using comfyUI as backend for my image generation service  and glad to see this repo.
I am just beginner of runpod and docker, hope it will be nice for my application.
Thank the great work of Blibla.

**Comment by drvpn** [comment]
Hello!
I am attempting to use this image to bake models into the image.  That seems to work except occasionally when running a request it tries to download a model?  WHY?  Below normally takes 1-2 seconds but randomly it takes 30 or 40!  I would LOVE to be able to bake models into an image and then use that specific baked in model UNTIL THE END OF TIME!  How can I make this happen?  PLEASE help!  Thanks! 

loaded partially 10210.496 10209.673889160156 0
100%|██████████| 60/60 [01:20<00:00,  1.34s/it]

**Reply by pribeh** [reply]
I'm attempting to do the same. any tips would be appreciated

**Comment by ZainCode20** [comment]
Helpful Repo
